src/options.c

```diff
@@ -262,31 +262,27 @@ static struct opt_entry opt_list[] = {
 "--single-wordlist=FILE     *short* wordlist with static seed words/morphemes\n" \
 "--single-user-seed=FILE    wordlist with seeds per username (user:password[s]\n" \
 "                           format)\n" \
-"--[no-]single-retest-guess override config for SingleRetestGuess\n" \
 "--single-pair-max=N        override max. number of word pairs generated (%u)\n" \
 "--no-single-pair           disable single word pair generation\n" \
+"--[no-]single-retest-guess override config for SingleRetestGuess\n" \
 "--wordlist[=FILE] --stdin  wordlist mode, read words from FILE or stdin\n" \
 "                  --pipe   like --stdin, but bulk reads, and allows rules\n" \
-"--mem-file-size=SIZE       size threshold for wordlist preload (default %u MB)\n" \
-"--loopback[=FILE]          like --wordlist, but extract words from a .pot file\n" \
-"--dupe-suppression         suppress all dupes in wordlist (and force preload)\n" \
-FUZZ_USAGE \
-PRINCE_USAGE \
-"--encoding=NAME            input encoding (eg. UTF-8, ISO-8859-1). See also\n" \
-"                           doc/ENCODINGS.\n" \
 "--rules[=SECTION[,..]]     enable word mangling rules (for wordlist or PRINCE\n" \
-"                           modes), using default or named rules\n" \
```

Comment:
No wait, this line went astray!

Reply:
Just fixed in a new commit that I squashed.